src/System.Linq.Expressions/src/System/Runtime/CompilerServices/ReadOnlyCollectionBuilder.cs

```diff
@@ -219,11 +221,11 @@ public void Clear()
         /// <returns>true if item is found in the <see cref="ReadOnlyCollectionBuilder{T}"/>; otherwise, false.</returns>
         public bool Contains(T item)
         {
-            if ((Object)item == null)
+            if ((object)item == null)
```

Comment:
IIRC, the JIT will avoid the boxing operation in this case for T == value type, right?